cmake/modules/Buildpmem.cmake

```diff
@@ -11,6 +11,17 @@ function(build_pmem)
   endif()
   set(PMDK_LIB "${PMDK_SRC}/${PMDK_LIB_DIR}")
 
+  find_program(MAKE_EXECUTABLE NAMES gmake make)
```

Comment:
Nit: this section seems to be copied. Can it not just be defined once and used throughout when building external libraries?

Reply:
good point, will extract this into a cmake function.

Reply:
@dillaman updated and repushed.